datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -0,0 +1,553 @@
+package datadog
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+)
+
+func resourceDatadogSecurityMonitoringRule() *schema.Resource {
+	return &schema.Resource{
+		Exists: resourceDatadogSecurityMonitoringRuleExists,
```

Comment:
You should remove the exists, and check for 404 in read instead (and set the id to "" when you get a 404)

Reply:
Done in the last commit, can you double check how I did it though? I didn't find any other resources which omit `exists`.